cpp/micro_benchmarks/mixtureOfExpertsBackendBenchmarkLauncher.cu

```diff
@@ -497,10 +497,10 @@ void argGenHardcoded(benchmark::internal::Benchmark* benchmark)
     auto inter_size_mul = {4.f};               // {7.f/2.f, 4.f};
     auto num_tokens = {2048};                  // {1, 20, 200, 2048};
     auto use_bias = {0};                       // {0, 1};
-    auto activation_type = {tensorrt_llm::ActivationType::Gelu};
-    // {tensorrt_llm::ActivationType::Relu, tensorrt_llm::ActivationType::Gelu,
-    // tensorrt_llm::ActivationType::Silu, tensorrt_llm::ActivationType::Geglu,
-    // tensorrt_llm::ActivationType::Swiglu};
+    auto activation_type = {ActivationType::Gelu};
```

Comment:
FYI line 520 and 543 are missing the mUseFinalScale parameter, do you mind updating them while you are here?
Just pass `1` on 520